### ISSUE TITLE
Configure statsd hostname and port

### ISSUE THREAD

**Issue by Dids** [realm/realm-object-server#346]
## Goals

I wanted to change the _statsd_ hostname/port that Realm Object Server is sending metrics to, but didn't find any documentation or configuration related to this, even after lightly combing through the source of the npm module(s).

The documentation only mentions that it expects _statsd_ to be listening on `localhost:8125`.

## Expected Results

Both hostname and port should be configurable while configuring Realm Object Server.

**Comment by astigsen** [comment]
@nirinchev @fealebenpae can one of you comment on this?

**Comment by fealebenpae** [comment]
ROS 3.x doesn't officially support sending metrics to a statsd host. There is a sub-component (the sync worker) that reports its own metrics in a statsd format, but ROS doesn't expose a configuration to override the endpoint it connects to. Instead, the sync worker's, and others, metrics are available via the `StatsSink` API.

**Comment by ianpward** [comment]
@kraenhansen Doesn't your work here allow for statsd on all ROS components now ? Everything that was part of Prometheus is now emitted with statsd ya?

**Comment by kraenhansen** [comment]
@ianpward that is the purpose of https://github.com/realm/realm-object-server-private/issues/1134 which I will be starting shortly.

**Comment by kraenhansen** [comment]
I would love to know @Dids - what tools would you like these metrics to be emitted into? Our current route would be to test with https://github.com/influxdata/telegraf - but do you have a tool that you will be using to consume these metrics?

**Comment by Dids** [comment]
I've actually recently migrated everything to Realm Cloud, so no longer personally need custom metrics.

Originally, I'm not sure if it was on ROS 2.x or 3.x, I saw that the metrics were being exported to the current host (hardcoded to 0.0.0.0 or 127.0.0.1, can't remember which), and since I ran ROS inside a Docker container, this wasn't an option.

If nobody else needs this kind of functionality, feel free to close the issue.
